samples/Grafana/19297_rev1.json

```diff
@@ -0,0 +1,1201 @@
+{
+  "__inputs": [
+    {
```

Comment:
This is the actual deployed Grafana dashboard right? Maybe it shouldn't be under samples?